db.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+const _ = require('lodash');
+const pg = require('pg');
+const format = require('pg-format');
+const moment = require('moment');
+
+const dbConfig = {
+  database: 'ripta'
+};
+
+const pool = new pg.Pool(dbConfig);
+
+const query = (sql) => {
+  const cleanedQuery = format(sql);
+
+  return new Promise((resolve, reject) => {
+    pool.query(cleanedQuery, (err, result) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(result.rows);
+      }
+    });
+  });
+};
+
+const getScheduledArrivalTS = (trip_id, stop_id) => {
```

Comment:
For example, this method and most of the methods below would be model methods off of a `Trip` model